lib/pfconfig/namespaces/resource/UriFilters.pm

```diff
@@ -1,44 +1,33 @@
-package pfconfig::namespaces::resource::Profile_Filters;
-
+package pfconfig::namespaces::resource::UriFilters;
 =head1 NAME
 
-pfconfig::namespaces::resource::Profile_Filters
+pfconfig::namespaces::resource::UriFilters
```

Comment:
I think it should be URI_Filter :)
It is more readable and usually singular is better than plural. :nail_care: